### ISSUE TITLE
Support deserializing integer geometries

### ISSUE THREAD

**Issue by michaelkirk** [georust/wkt#102]
#101 added support for serializing integer geometries to WKT, but I hit an issue while implementing deserialization for integer types.

My WIP is here:
https://github.com/georust/wkt/tree/mkirk/try-from-non-floats

**Comment by rmanoka** [comment]
Err-ing if the scalar can't be parsed feels like the right thing to do;  agree that the error message could be more helpful.  I don't think there's a huge use-case for this, so we could wait and hear more thoughts.  Thanks for all the effort!

**Comment by urschrei** [comment]
I agree that `Err` is the way to go here – with a more informative error, its resolution can be left up to the library user.
